README.Rmd

```diff
@@ -102,6 +102,7 @@ The rest of this documentation includes:
   notes and suggestions for running `growthcleanr` with large data sources
 - [Next steps](https://carriedaymont.github.io/growthcleanr/articles/next-steps.html),
   notes on potential enhancements to the pediatric and adult algorithms
+- [Developer Guidelines](https://carriedaymont.github.io/growthcleanr/articles/developer-guidelines.html), advice for contributors to this package, including a CRAN release checklist
```

Comment:
My editor-brain wants to lower case "Guidelines" in the link text for consistency with the others.

Reply:
done!